src/webapp/pages/about/AboutPage.tsx

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import i18n from "../../../locales";
+import { IconButton, Typography } from "@material-ui/core";
+import HomeIcon from "@material-ui/icons/Home";
+import { NavLink } from "react-router-dom";
+import { useGoBack } from "../../hooks/useGoBack";
+
+export const AboutPage = () => {
+    const goBack = useGoBack();
+    return (
+        <Container>
+            <Wrapper>
+                <BackButton
+                    title="Go to dashboard"
+                    onClick={goBack}
+                    color="inherit"
+                    aria-label={i18n.t("Back")}
+                    data-test={"page-header-back"}
+                >
+                    <HomeIcon fontSize="small" color="inherit" />
+                </BackButton>
+                <Title variant="h1">About Sharing Settings App</Title>
```

Comment:
Should be `i18n.t("About Sharing Settings App")`. Unless we don't need the translations here for any reason?

Reply:
Right! I missed that one